vault/data_source_generic_secret_test.go

```diff
@@ -162,6 +163,24 @@ func testDataSourceGenericSecret_check(s *terraform.State) error {
 		return fmt.Errorf("resource has no primary instance")
 	}
 
+	ts, ok := iState.Attributes["lease_start_time"]
+	if !ok {
+		return fmt.Errorf("lease_start_time not set")
+	}
+
+	t, err := time.Parse(time.RFC3339, ts)
+	if err != nil {
+		return fmt.Errorf("lease_start_time value %q is not in the expected format, err=%s", ts, err)
+	}
+
+	elapsed := time.Now().UTC().Unix() - t.Unix()
+	// give a reasonable amount of buffer to allow for any system contention.
+	maxElapsed := int64(30)
+	if elapsed > maxElapsed {
+		return fmt.Errorf(
+			"elapsed lease_start_time %ds exceeds maximum %ds", elapsed, maxElapsed)
```

Comment:
nit: newline here looks kind of surprising given the slightly longer line above. Are you wanting to enforce a strict line length limit?

Reply:
yes, set my IDE to 100 chars max - I think the original message exceeded that.

Fixed in 85d95f3e4885ee78883e1bbcb5a99c78262d30a0